test/parallel/test-util-inspect.js

```diff
@@ -870,7 +870,7 @@ if (typeof Symbol !== 'undefined') {
   const set = new Set(['foo']);
   set.bar = 42;
   assert.strictEqual(
-    util.inspect(set, true),
+    util.inspect(set, { showHidden: true }),
```

Comment:
does this make this semver-major? (even though `true` as second arg wasn't documented?)

Reply:
Oh, it's just an alternative way of writing it. It's an unrelated change that I can revert. It's just better to read for me.

Reply:
NM, it's still covered by https://github.com/nodejs/node/blob/2b1c2a46783a9496e4559e6dfe15bf7ef7cf19cd/test/parallel/test-util-inspect.js#L77